### DISCUSSION TITLE
How can `collapse`/`expand` all function inside a module?

### DISCUSSION THREAD

**Discussion by shahryarjb** [zed-industries/zed#10036] [Feature Requests]
Hi, is there any option to `collapse` or `expand` all functions of a module/file?
Thanks

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
